docs/api/supervisor/endpoints.md

```diff
@@ -761,11 +761,12 @@ Create a full backup.
 
 **Payload:**
 
-| key        | type    | optional | description                              |
-| ---------- | ------- | -------- | -----------------------------------------|
-| name       | string  | True     | The name you want to give the backup     |
-| password   | string  | True     | The password you want to give the backup |
-| compressed | boolean | True     | `false` to create uncompressed backups   |
+| key        | type           | optional | description                                    |
+| ---------- | -------------- | -------- | ---------------------------------------------- |
+| name       | string         | True     | The name you want to give the backup           |
+| password   | string         | True     | The password you want to give the backup       |
+| compressed | boolean        | True     | `false` to create uncompressed backups         |
+| location   | string or null | True     | Name of a backup mount or `null` for /backup   |
```

Comment:
Should we call this out as “default location (/config)”?

Reply:
That makes sense. Was going for brevity but probably too short

Reply:
er wait, not "default location" since that may no longer be `/backup`. `null` is used specifically when you want to put the backup in the local `/backup` folder. If you want to use whatever is set as the default backup mount then you omit `location`